src/boots/boots.gd

```diff
@@ -0,0 +1,45 @@
+extends Node2D
+
+
+var sequence_to_reach_golden_piece: Array[String] = ["down","down","down","down","down"]
+var past_moves_relevant_to_code: Array[String] = ["","","","",""]
+
+# Called when the node enters the scene tree for the first time.
+
+func _ready() -> void:
+	print("BOOTS ON!!!")
+	connect_move_signal_to_player()
+
+func connect_move_signal_to_player() -> void:
+	var parent_player_node = get_node("../")
+	# Yeah!! You can get stuff 2 parents up the tree get_node("../../")
+	if parent_player_node != self:
```

Comment:
using get_parent() when you're the root/have no parent would return the "root"/window godot node, and the level above that is nothing

so it would never be itself? what is this check for?

Reply:
1 time safety measure, don't want any weird unexpected connections. If you don't want it, you can remove, I just think in case the scene is tested alone or such